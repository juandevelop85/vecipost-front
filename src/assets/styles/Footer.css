.footer {
  width: 100%;
  height: 50px;
  border-top: 1px solid #eaeaea;
  display: flex;
  justify-content: center;
  align-items: center;
  background-color: #20377c;
  color: white;
  position: fixed;
  left: 0;
  bottom: 0;
  z-index: 100;
}

.footer > div {
  margin: 10px;
  padding: 20px;
  font-size: 30px;
  width: 33.33%;
  text-align: start;
  display: grid;
  float: left;
}


.footer img {
  margin-left: 0.5rem;
}

.footer a {
  display: flex;
  justify-content: center;
  align-items: center;
}
