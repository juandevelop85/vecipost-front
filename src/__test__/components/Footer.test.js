import React from 'react';
import { mount } from 'enzyme';
import { create } from 'react-test-renderer';
import Footer from '../../components/Footer';

describe('<Footer />', () => {
  const footer = mount(<Footer />);
  test('Render del componente Footer', () => {
    expect(footer.length).toEqual(1);
  });
  test('Render del titulo', () => {
    expect(footer.find('.footer').text()).toEqual('Juan Vernaza con ❤️');
  });
});

describe('Footer snapshot', () => {
  test('Comprobar la UI del componente Footer', () => {
    const footer = create(<Footer />);
    expect(footer.toJSON()).toMatchSnapshot();
  });
});
