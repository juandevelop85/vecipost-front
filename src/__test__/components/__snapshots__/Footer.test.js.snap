// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot Comprobar la UI del componente Footer 1`] = `
<div
  className="footer"
  id="parent"
>
  <div
    className="row center full"
    id="child"
  >
    <p
      className="Footer-title"
    >
      Juan Vernaza con ❤️
    </p>
  </div>
</div>
`;
